.vscode/settings.json

```diff
@@ -13,24 +8,11 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "[javascript]": {
-    "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  },
-  "[json]": {
-    "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  },
   "python.formatting.provider": "black",
```

Comment:
I'm pretty sure this is what tells VS Code to use `black` - see comment above.

Reply:
Correct, but it appears the extension is needed as well.